code/modules/mob/living/simple_animal/hostile/abilities_common.dm

```diff
@@ -105,6 +107,9 @@
 	immune_to_lowpop = TRUE
 	allowed_mobs = RTS_RATS_ALLOWED
 
+/obj/effect/proc_holder/mob_common/summon_backup/rat/tame
+	allowed_mobs = RTS_FRATS_ALLOWED
+
 /obj/effect/proc_holder/mob_common/summon_backup/robot
 	allowed_mobs = RTS_ROBOT_ALLOWED
 	banned_from_lowpop = TRUE
```

Comment:
Think I found why Summon Backup didnt work with humans, its the check on 118, and the var on 122. If you change if(!istype(user, /mob/living/simple_animal)) to if(!istype(user, /mob)) and var/mob/living/simple_animal/owner = user to var/mob/owner = user that should make it work on humans. =3

Reply:
Thank you! I'll tweak it and some other stuff shortly.